evennia/commands/default/system.py

```diff
@@ -1203,3 +1205,183 @@ def func(self):
                 "*" if sub[5] else "-",
             )
         self.caller.msg("|wActive tickers|n:\n" + str(table))
+
+
+class CmdTasks(COMMAND_DEFAULT_CLASS):
+    """
+    Manage active tasks (delays).
+
+    Note:
+      Only the action requested on the first switch will be processed. All other switches will
+      be ignored.
+
+      Manipulation of a single task is intended to be done via the clickable links or through
+      code directly. Due to generally short life of a task, the inclusion of completion date
+      and function's memory reference guarentees an incorrect task will not be manipulated.
+
+      By default, tasks that are canceled and never called are automatically removed after
+      one minute.
+
+    Usage:
+      tasks
+        show all current active taks.
+      tasks[/switch] [function name]
+        Process the action in the switch to all tasks that are deferring a specific function name.
+        This would match the name of the callback you passed to a delay or the task handler.
+      tasks[/switch] [task id], [completion date], [function memory reference]
+        Process the action in the switch to a specific task.
```

Comment:
This is also the in-game help for this command, so look at how these sections are ordered for other default commands:

```
Short intro

Usage:
    tasks[/switch] [task_id][, completion_date[, functionref]

Switches:

Notes:

Example:

```

Reply:
I do get a cyclomatic warning on func. I did put in line breaks at major logic changes.
I understand the width logic now. Did replace collection of screen width with client_width.
I changed my linter width warning to 100 and resolve the items that were longer than 100.
I am going to continue forward and commit once completed with the remaining requests.